mollie/api/resources/base.py

```diff
@@ -21,55 +19,30 @@ def get_resource_object(self, result):
     def get_resource_name(self):
         return self.__class__.__name__.lower()
 
-    def rest_create(self, data, params=None):
+    def create(self, data=None, **params):
         path = self.get_resource_name()
         result = self.perform_api_call(self.REST_CREATE, path, data, params)
         return self.get_resource_object(result)
 
-    def rest_read(self, resource_id, params=None):
+    def get(self, resource_id, **params):
         path = self.get_resource_name() + '/' + str(resource_id)
         result = self.perform_api_call(self.REST_READ, path, params=params)
         return self.get_resource_object(result)
 
-    def rest_update(self, resource_id, data, params=None):
+    def update(self, resource_id, data=None, **params):
         path = self.get_resource_name() + '/' + str(resource_id)
         result = self.perform_api_call(self.REST_UPDATE, path, data, params)
         return self.get_resource_object(result)
 
-    def rest_delete(self, resource_id, params=None):
+    def delete(self, resource_id):
```

Comment:
Afaik, we don't support query string parameters in the `DELETE` requests that our API offers.

Reply:
Updated this to support request bodies (`data` parameter).